addons/project/views/project_views.xml

```diff
@@ -960,16 +981,31 @@
                         <page name="task_dependencies" string="Blocked By" attrs="{'invisible': [('allow_task_dependencies', '=', False)]}" groups="project.group_project_task_dependencies">
                             <field name="depend_on_ids" nolabel="1" context="{'default_project_id' : project_id}">
                                 <tree editable="bottom">
+                                    <field name="legend_normal" invisible="1"/>
+                                    <field name="legend_done" invisible="1"/>
+                                    <field name="legend_blocked" invisible="1"/>
+                                    <field name="message_needaction" invisible="1" readonly="1"/>
                                     <field name="is_closed" invisible="1" />
-                                    <field name="name" />
+                                    <field name="allow_subtasks" invisible="1" />
+                                    <field name="sequence" invisible="1" readonly="1"/>
```

Comment:
This field seems useless in this subview, since we cannot normally reorder the subtasks. 🤔